Makefile

```diff
@@ -0,0 +1,97 @@
+# By default, let's print out some help
+.PHONY: usage
+usage:
+	@echo "$$(tput bold)Welcome to libtock-rs!$$(tput sgr0)"
+	@echo
+	@echo "First things first, if you haven't yet, check out Tocks's doc/Getting_Started."
+	@echo "After that read the README from libtock-rs"
+	@echo "You'll need to install a few requirements before we get going."
+	@echo
+	@echo "The next step is to choose a board to build Tock for. Mainline"
+	@echo "libtock-rs currently includes support for the following platforms:"
+	@echo " - hail"
+	@echo " - nrf52840"
+	@echo " - opentitan"
+	@echo " - hifive1"
+	@echo " - nrf52"
```

Comment:
What I really like about this PR is that the platform names have been cleaned up. Before, it was a mixture of CPU architectures and board layouts.

Reply:
Exactly!  :)